azurerm/resource_arm_storage_account.go

```diff
@@ -963,3 +972,124 @@ func flattenAzureRmStorageAccountIdentity(identity *storage.Identity) []interfac
 
 	return []interface{}{result}
 }
+
+func flattenAzureRmStorageAccountPrimaryEndpoints(d *schema.ResourceData, primary *storage.Endpoints, acctName *string, acctKey *string) error {
+	if primary == nil {
```

Comment:
If we do this non of the properties will be set to empty and the properties won't exist. This check should be inverted.

Reply:
If the primary endpoint is nil, then it's an API error. Do you think that if we need to silent the error or report it?